src/components/avatar/avatar.ts

```diff
@@ -27,7 +27,14 @@ export class MAvatar extends Vue {
     @Prop({ default: false })
     clickable: boolean;
 
-    hover: boolean = false;
+    isFocusVisible: boolean = false;
+    isHovered: boolean = false;
+    isTouched: boolean = false;
+    timeoutContentVisible: any = undefined;
+
+    created(): void {
+        document.addEventListener('click', this.onDocumentClick);
```

Comment:
Bizz qu'on ai besoin de faire cela, car, les cas ou l'event click est prevent par un autre component ou si on click en dehord de la fenetre cette fonctionne ne sera pas appelé.  Aussi j'aime pas trop le fait d'ajouter des évènemnt sur document , a mettons qu'on a 2000 avatars ca fait 2000 event listeners 

Mais bon je comprend que c'est pour l'effet de survol en mobile seulement.. je vais en parlé avec @jipigi

Reply:
On pourrait utiliser l'événement de $modul en utilisant le plugin de modul.
this.$modul.event.$on('click', this.onDocumentClick);
Vue.use(ModulPlugin);

Reply:
En fait j'aurais pu mettre un "ontouchend" probablement. Sinon on a aussi le timeout qui ferme automatiquement le hover alors je peux en laisser seulement un. À vous de voir! :)

Reply:
Une bonne idée de @Mboulianne, créer le listener uniquement lorsque le hover est activer et le remove lorsqu'il perd le focus.